es6-shim.js

```diff
@@ -1844,13 +1844,7 @@
 
     atanh: function atanh(value) {
       var x = Number(value);
-      if (Number.isNaN(x) || x < -1 || x > 1) {
-        return NaN;
-      }
-      if (x === -1) { return -Infinity; }
-      if (x === 1) { return Infinity; }
-      if (x === 0) { return x; }
-      return 0.5 * _log((1 + x) / (1 - x));
+      return Math.sign(x) * Math.log1p(2 * Math.abs(x) / (1 - Math.abs(x))) / 2;
```

Comment:
only readability, but that's subjective, and i'd argue that using `Math.sign` and having it point not at the global `Math.sign` would make the code harder to understand - this way, grepping for `_sign` is pretty straightforward.